config/settings.yml

```diff
@@ -972,7 +972,7 @@
   :prefetch_stale_threshold: 30.seconds
   :rails_server: puma
   :remote_console_type: VMRC
-  :role: database_operations,event,reporting,scheduler,smartstate,ems_operations,ems_inventory,user_interface,websocket,web_services,automate
+  :role: database_operations,event,reporting,scheduler,smartstate,ems_operations,ems_inventory,user_interface,websocket,web_services,automate,smartproxy
```

Comment:
Is this turning the role on by default?

Reply:
That line shouldn't be in there.  Only the timeout below was meant to be changed.  Thanks for noticing.